ledger/participant-integration-api/src/main/scala/platform/store/backend/common/ContractStorageBackendTemplate.scala

```diff
@@ -52,6 +52,8 @@ trait ContractStorageBackendTemplate extends ContractStorageBackend {
       s"The following contracts have not been found: ${missingContractIds.map(_.coid).mkString(", ")}"
     )
 
+  protected val CastNullLedgerEffectiveTime: String = "NULL::BIGINT"
```

Comment:
@nmarton-da What do you think about defining this and overriding in subclasses?

Reply:
how about just adding to querystrategy with some more general name like nullAsBigint?